product_harmonized_system_per_country/models/product_product.py

```diff
@@ -0,0 +1,39 @@
+#  Copyright (c) 2024 Groupe Voltaire
+#  @author Emilie SOUTIRAS  <emilie.soutiras@groupevoltaire.com>
+#  License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
```

Comment:
This class is in product_product.py, It will be clearer in a dedicated file product_template.py (or rename the file as product.py but I prefer the first option)